browser/android/preferences/brave_pref_service_bridge.cc

```diff
@@ -268,6 +268,12 @@ void JNI_BravePrefServiceBridge_SetReferralDownloadId(
       kReferralDownloadID, ConvertJavaStringToUTF8(env, downloadId));
 }
 
+jboolean JNI_BravePrefServiceBridge_IsBraveRewardsEnabled(
```

Comment:
This isn't needed and we should stop creating individual methods for these because we can access prefs directly in android, see NTP_SHOW_SPONSORED_IMAGES_BACKGROUND_IMAGE
cc @SergeyZhukovsky